src/workerd/api/node/zlib-util.c++

```diff
@@ -397,7 +397,7 @@ kj::Maybe<CompressionError> ZlibContext::setParams(int _level, int _strategy) {
   return kj::none;
 }
 
-void ZlibContext::close() {
+ZlibContext::~ZlibContext() {
```

Comment:
I see these are marked as `KJ_DISALLOW_COPY` but what if these are `kj::mv`'d? Will the destructor on the moved-from original be safe? Or should these be `KJ_DISALLOW_COPY_AND_MOVE`?

Reply:
You're right. I've added them all. Thank you!